### PR TITLE
✅ Add post mortem for full Loki PVC situation

### PR DESCRIPTION
**Pull request by deifyed**
## Description

Adds a post mortem for the full Loki PVC problem

### CODE REVIEW

File: docs/post-mortems/0005-loki-pvc-full.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+## Impact
+<!--
+Describe the consequences this had for the organization.
+
+Example:
+- Approximately 30.000 users downloaded and experienced an error using the app.
+- Users already logged in experienced some minor delay in the app
+-->
+- Citizens expecting a mail regarding kindergarden benefits experienced an ~eight~ hour delay?
```
Comment (olovholm) [inline docs/post-mortems/0005-loki-pvc-full.md:49]: - During the annual creation of re-applications for citizens with reduced payment the processing had to be halted for ~eight~ hours. 
- Approximately two hours of log data got lost.

Comment (olovholm) [inline docs/post-mortems/0005-loki-pvc-full.md:49]: - During the annual creation of re-applications for citizens with reduced payment the processing had to be halted for ~eight~ hours. 
- Approximately two hours of log data got lost.

File: docs/post-mortems/0005-loki-pvc-full.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Impact
+<!--
+Describe the consequences this had for the organization.
+
+Example:
+- Approximately 30.000 users downloaded and experienced an error using the app.
+- Users already logged in experienced some minor delay in the app
+-->
+- During the annual creation of re-applications for citizens with reduced payment the processing had to be halte for
```
Comment (yngvark) [inline docs/post-mortems/0005-loki-pvc-full.md:49]: halte -> halted

**Review by yngvark — APPROVED** [review]
Looks good!

File: docs/post-mortems/0005-loki-pvc-full.md (hunk 2 above)
Comment (deifyed) [inline docs/post-mortems/0005-loki-pvc-full.md:49]:
```suggestion
- During the annual creation of re-applications for citizens with reduced payment the processing had to be halted for
```

**Pull request merged**